test/test_checks.py

```diff
@@ -33,11 +33,10 @@ def test_stdin(self):
 
         stdout_lines = stdout.splitlines()
         self.assertEqual(stderr, b'')
-        self.assertEqual(stdout_lines, [
-            b'stdin:1:25: Q000 Remove bad quotes.',
-            b'stdin:2:25: Q000 Remove bad quotes.',
-            b'stdin:3:25: Q000 Remove bad quotes.',
-        ])
+        self.assertEqual(len(stdout_lines), 3)
```

Comment:
I think it would be better if there isn't a separate line count check before. If the output is different it might just tell the difference in line numbers, which is not really helpful. So I'd do this last, but then you might not know how the other lines looked when it fails with the first line. I wouldn't be concerned that it might compare a line that doesn't exist, because that indirectly is the same as this check (with the only difference that this check also fails when the number of lines is to high).

Not sure what is the best way here.